python/paddle/base/framework.py

```diff
@@ -7234,19 +7203,11 @@ def set_state_dict(self, state_dict, scope=None):
                 try:
                     vars_dict[name].set_value(value, scope)
                 except ValueError as err:
-                    warnings.warn(
-                        "Skip loading for '{}'. ".format(name) + str(err)
-                    )
+                    warnings.warn(f"Skip loading for '{name}'. " + str(err))
                 except TypeError as err:
-                    warnings.warn(
-                        "Skip loading for '{}'. ".format(name) + str(err)
-                    )
+                    warnings.warn(f"Skip loading for '{name}'. " + str(err))
             else:
-                warnings.warn(
-                    "Skip loading for '{0}'. Because '{0}' not in the program.".format(
-                        name
-                    )
-                )
+                warnings.warn(f"Skip loading for '{name}'. Because '{name}' not in the program.")
```

Comment:
```suggestion
                warnings.warn(
                    f"Skip loading for '{name}'. Because '{name}' not in the program."
                )
```